### ISSUE TITLE
Correct time format

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/openstreetmap-website#2603]
This changeset was made a minute ago:

![time](https://user-images.githubusercontent.com/57260466/80280243-2e520400-8703-11ea-8a6a-54901a2229da.png)

and the time in my country was 14:22.
This is correct because there is also "+0000" which gives me information that the time is UTC.

But on the messages page there is the same time but without information that this is UTC time and not my local time.

So if you're not planning to change the time according to the user's time zone, please at least add "+0000" or "UTC" to the time we can see on the messages page. There is only 1 or 2 hours difference between UTC and Central Europe timezone, but I suppose users in America see the same UTC time so they don't know at what time they got a message.

**Comment by tomhughes** [comment]
Same answer as for #2602 I'm afraid - you will need to get this fixed in rails-i18n after which it will come through to here automatically as we update.

**Comment by tomhughes** [comment]
Oh hang on, this is just about the time zone, not the actual formatting? So the title of is misleading?

**Comment by tomhughes** [comment]
There's no way for us to know the time zone of a user so there's not really a huge amount we can do here.

We could explicitly indicate UTC wherever a time appears but it does make things rather verbose.

**Comment by maro-21** [comment]
> Oh hang on, this is just about the time zone, not the actual formatting? So the title is misleading?

No. I meant to change the dates format in the messages page and add +0000 or UTC as it is in the osm.org/history.

**Comment by tomhughes** [comment]
Right but that amounts to the same thing - you just want a format that shows the time zone.

**Comment by dieterdreist** [comment]
sent from a phone

> On 25. Apr 2020, at 15:41, Tom Hughes <notifications@github.com> wrote:
> 
> There's no way for us to know the time zone of a user so there's not really a huge amount we can do here.
> 


Preferred timezone could become a user setting.
Maybe it could be looked up from the home location which is already a user setting

**Comment by jonycoo** [comment]
@tomhughes 
sorry I have a connected comment.
I'm using the OSM API v0.6.
When getting elements it uses the ISO 8601 formatted date,
when getting notes and comments it uses 2018-11-14 22:07:37 UTC this 
non ISO 8601 format.
I suggest to the server developers use one common format.

**Comment by simonpoole** [comment]
> 
> 
> @tomhughes
> sorry I have a connected comment.
You should be opening a separate issue for a separate topic.

> I'm using the OSM API v0.6.
> When getting elements it uses the ISO 8601 formatted date,
> when getting notes and comments it uses 2018-11-14 22:07:37 UTC this
> non ISO 8601 format.
> I suggest to the server developers use one common format.

The notes API is an implementation of an older, originally separate, API, and was kept backwards compatible with the original implementation. A change (which I would support, as I've moaned about this issue before too), would require a major version change (as it wouldn't be backwards compatible), which is unlikely to happen "just because of this annoyance".

**Comment by maro-21** [comment]
This can be really easily fixed, just add UTC to the time and date.
Currently on the messages page time is "4 June 2020 20:11" but when I'm replying to a message the date changes to "On 2020-06-04 20:11:33 UTC someone wrote:". So you can just copy that string I think.
